src/synthesizer/particle/galaxy.py

```diff
@@ -1486,19 +1447,134 @@ def make_ssfr_map(
         Returns:
             Image
         """
-
         # Get the SFR map
-        img = self.make_sfr_map(
+        img = self.get_map_sfr(
             resolution=resolution,
             fov=fov,
             img_type=img_type,
-            cosmo=cosmo,
             kernel=kernel,
             kernel_threshold=kernel_threshold,
             age_bin=age_bin,
         )
 
         # Convert the SFR map to sSFR
-        img.img /= self.stellar_mass
+        img.arr /= self.stellar_mass.value
+        img.units = img.units / self.stellar_mass.units
 
         return img
+
+    def get_data_cube(
+        self,
+        resolution,
+        fov,
+        lam,
+        cube_type="hist",
+        stellar_spectra=None,
+        blackhole_spectra=None,
+        kernel=None,
+        kernel_threshold=1,
+        quantity="lnu",
+    ):
+        """
+        Make a SpectralCube from an Sed.
```

Comment:
shouldn't this be from a `galaxy` object?

Reply:
Should what be from a galaxy object? Sorry, I don't understand what you mean from the highlighted code.

Reply:
Clarified now.